addons/calendar/models/ir_http.py

```diff
@@ -12,7 +12,7 @@ class IrHttp(models.AbstractModel):
 
     @classmethod
     def _auth_method_calendar(cls):
-        token = request.params.get('token', '')
+        token = request.get_http_params().get('token', '')
```

Comment:
`request.params` is the black sheep of the design and we don't have a clean solution yet, we decided with my team to leave this point open in order to merge httpocalypse. A property wouldn't work well because we need to inject the args from the URL (4 in `/record/4`) that we only know after `match()`, also we must keep in mind that `request.params` in a JSON controller is the json-deserialized body. I totally agree that `get_http_params()` isn't great but at least it is explicit and *explicit is better than implicit*.